src/loaders/FileLoader.js

```diff
@@ -88,6 +102,12 @@ class FileLoader extends Loader {
 
 					}
 
+					if ( isRangeRequest && response.status === 200 ) {
```

Comment:
This was metnioned in the upstream PR as well. I think its a bit odd to throw in this case. The thing using the FileLoader should decide what they want to do in this case. Ex. The GLBRangeRequest should take that full response and pass it along to GLTFLoader so the whole glb does not need to be downloaded again.

With the above suggestion about cachekey it would also make it so this PR has nothing specific about range headers and is instead just fixing cache keys to respect headers.

Reply:
See https://github.com/mrdoob/three.js/pull/24580#issuecomment-1250005693

And Mugen approves

Reply:
I forgot FileLoader does additional parsing and doesn't just pass you the response directly. Unfortunately that would make it hard to do what I was suggesting. As for browser cache.. Ideally we should be able to rely on that, but I think the browser isn't so great about honoring cache on large files.

Reply:
I'm not sure if I understand correctly what you suggested but

> The thing using the FileLoader should decide what they want to do in this case.

This is possible. See

* https://github.com/mrdoob/three.js/pull/24580#issuecomment-1291195970
* https://github.com/takahirox/three-gltf-extensions/pull/90